src/app/components/hris/employees/employee-profile/employee-profile.component.ts

```diff
@@ -30,6 +30,8 @@ import { EmployeeTermination } from 'src/app/models/hris/employeeTermination.int
 import { Subscription } from 'rxjs';
 import { ClientService } from 'src/app/services/hris/client.service';
 import { SharedPropertyAccessService } from 'src/app/services/hris/shared-property-access.service';
+import * as pako from 'pako'; 
+import { ImageProcessingService } from 'src/app/services/hris/image-processing.service';
```

Comment:
Refactor: pako import should not be used here, only file-processing service import should be used.